File: Assignment33_2.c
#include <stdio.h>
#include <stdbool.h>

///////////////////////////////////////////////////////////////////////////////
// Function Name : ChkCapital
// Description   : Check whether given character is capital alphabet (A-Z)
// Input         : Character
// Output        : true / false
// Author        : Raj Samir Jadhav
// Date          : 28/11/2025
///////////////////////////////////////////////////////////////////////////////
bool ChkCapital(char ch)
{
    if(ch >= 'A' && ch <= 'Z')                                  // Check if character is between A and Z
    {
        return true;
    }
    else
    {
        return false;
    }
}

int main()
{
    char cValue = '\0';
    bool bRet = false;

    printf("Enter the character : ");
    scanf("%c", &cValue);

    bRet = ChkCapital(cValue);

    if(bRet == true)
    {
        printf("It is Capital Character\n");             // Displays if character is capital
    }
    else
    {
        printf("It is not a Capital Character\n");        // Displays if charcter is not Capital
    }

    return 0;
}

File: Assignment33_2.java
import java.util.Scanner;

///////////////////////////////////////////////////////////////////////////////
// Function Name : ChkCapital
// Description   : Check whether entered character is capital letter (A-Z)
// Input         : Character
// Output        : boolean TRUE / FALSE
// Author        : Raj Samir Jadhav
// Date          : 28/11/2025
///////////////////////////////////////////////////////////////////////////////
class Pattern
{
    public boolean ChkCapital(char ch)
    {
        if(ch >= 'A' && ch <= 'Z')                        // Check A to Z
        {
            return true;
        }
        else
        {
            return false;
        }
    }
}

class Assignment33_2
{
    public static void main(String[] args)
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the character : ");                       // Take input from user
        char cValue = sobj.next().charAt(0);

        Pattern pobj = new Pattern();
        boolean bRet = pobj.ChkCapital(cValue);

        if(bRet == true)
        {
            System.out.println("It is Capital Character");                    // Displays its capital Character
        }
        else
        {
            System.out.println("It is not a Capital Character");             // Displays if Character is not Capital
        }
    }
}
